website/docs/routes/traditional-media.mdx

```diff
@@ -302,6 +302,14 @@ The URL of the form `https://www.dnaindia.com/topic/dna-verified` demonstrates t
 
 </Route>
 
+## Klotener Anzeiger {#klotener-anzeiger}
+
+### News {#klotener-anzeiger-news}
+
+<Route author="JustYannicc" example="/kloteneranzeiger/news" path="/v2/kloteneranzeiger/news" radar="1" rssbud="1" paramsDesc="No parameters required. This route fetches the latest news articles from Klotener Anzeiger." radar="1"/>
+
+</Route>
```

Comment:
docs: https://docs.rsshub.app/joinus/new-rss/add-docs

Do not fill in `paramsDesc` property if the path does not include any path variables.

Don't write the `Route` component in both `self-closing tag` style and `a pair of tags style`.